x-pack/plugins/security_solution/common/ecs/file/index.ts

```diff
@@ -14,9 +14,30 @@ export interface CodeSignature {
   subject_name: string[];
   trusted: string[];
 }
+
+export interface Token {
+  integrity_level_name: string;
+}
+
+export interface MemoryPe {
+  imphash?: string;
+}
+
+export interface StartAddressDetails {
+  allocation_type?: string;
+  allocation_size?: number;
+  region_size?: number;
+  region_protection?: string;
+  memory_pe?: MemoryPe;
+}
```

Comment:
Do we want the rest of the fields from here too? https://github.com/elastic/endpoint-package/blob/1c983ab3fe1ebaa62171b38cb37c3dc420d9b16b/custom_schemas/custom_memory_region.yml

Reply:
@gabriellandau they are not used for the functionality in this PR so I left them out. I can add them.